pages.ko/common/basename.md

```diff
@@ -0,0 +1,11 @@
+# basename
+
+> 경로명의 디렉토리가 아닌 부분을 반환.
+
+- 경로에서 파일 이름만 표시:
+
+`basename {{path/to/file}}`

```

Comment:
Perhaps the generic ones here such as `path/to/file`, ` suffix`, `file1`, etc. could be translated too?

Reply:
I think it's more natural. but if you want, i will translate it

Reply:
Hrm. I think we should translate them. For example, take a look at this `pt_BR` page: https://github.com/tldr-pages/tldr/blob/master/pages.pt_BR/common/7za.md

Basically, anything that doesn't affect the execution of the command should be translated.

Reply:
@sbrl I translated them :)